src/txmempool.h

```diff
@@ -684,6 +684,8 @@ class CTxMemPool
         return (mapTx.count(gtxid.GetHash()) != 0);
     }
 
+    const CTxMemPoolEntry* GetEntry(const uint256& hash) const LIFETIMEBOUND EXCLUSIVE_LOCKS_REQUIRED(cs);
```

Comment:
6903d8aa37e01d16aea0b53c3729f8f92c7d2bf0
Could you use `Txid` instead? or `GenTxid`